package.json

```diff
@@ -69,6 +69,7 @@
     "axe-webdriverjs": "0.4.0"
   },
   "dependencies": {
+    "@okta/courage": "1.14.0",
```

Comment:
oh but that was what was generated by install-with-shrinkwrap. Should I manually update the optional-dependencies then?